cmd/virt-launcher/virt-launcher.go

```diff
@@ -444,6 +444,16 @@ func main() {
 		}
 	}
 
+	terminateIstioProxyIfPresent := func() {
+		if istioProxyPresent() {
+			err := exec.Command("curl", "-XPOST", "http://127.0.0.1:15020/quitquitquit").Run()
+			if err != nil {
+				log.Log.Error("Failed to request Istio proxy termination")
+			}
+		}
+	}
+	defer terminateIstioProxyIfPresent()
```

Comment:
I think this belongs in the ForkAndMonitor function.

Reply:
Thanks, Done